core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -38,7 +38,13 @@ object ConsoleConsumer extends Logging {
 
   def main(args: Array[String]) {
     val conf = new ConsumerConfig(args)
-    run(conf)
+    try {
+      run(conf)
+    } catch {
+      case e: Throwable =>
+        error("Error running consumer: ", e)
+    }
+    System.exit(0);
```

Comment:
@ijuma Thanks much for your comment. I have spent time thinking through the pros and cons of having braces. I finally decided to keep the braces, because 
- it doesn't add any additional line as compared to not having the brace
- the code style will be consistent with other parts of the Kafka code. It can be a little confusing if you have different code style even in the same file, e.g. ConsoleConsumer.

Please see updated patch. I have tested again to make sure the code solves the problem. Thanks!